Assets/ConsentManagementProvider/Scripts/facade/CMP.cs

```diff
@@ -26,13 +26,12 @@ internal static ISpSdk ConcreteInstance
             get
             {
                 if (concreteInstance == null)
-#if UNITY_ANDROID
-                    concreteInstance = new ConsentWrapperAndroid();
-#elif UNITY_IOS && !UNITY_EDITOR_OSX
-                    concreteInstance = new ConsentWrapperIOS();
-#else
-                    concreteInstance = new ConsentWrapperUnityEditor();
-#endif
+                    concreteInstance = Application.platform switch
+                    {
+                        RuntimePlatform.Android => new ConsentWrapperAndroid(),
+                        RuntimePlatform.IPhonePlayer => new ConsentWrapperAndroid(),
```

Comment:
> RuntimePlatform.IPhonePlayer => new ConsentWrapperAndroid()
@Nevazhnovu is this line correct?

Reply:
@andresilveirah you are right, that's a typo. My bad!